Nightly github actions failing due to broken recipe?

Hi,

I was wondering if the nightly builds are failing due to a broken recipe in the following recipe: `recipes/perl-template-toolkit`

The github action logs contain the following:
```
Could not solve for environment specs
The following packages are incompatible
├─ perl-image-info is not installable because it requires
│  └─ perl-xml-libxml, which does not exist (perhaps a missing channel);
└─ perl-test-leaktrace does not exist (perhaps a typo or a missing channel).

....
....

00:24:45 BIOCONDA ERROR COMMAND FAILED (exited with 1): /opt/mambaforge/envs/bioconda/bin/conda-build --override-channels --no-anaconda-upload -c conda-forge -c bioconda -e /opt/mambaforge/envs/bioconda/conda_build_config.yaml -e /opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/bioconda_utils/bioconda_utils-conda_build_config.yaml recipes/perl-template-toolkit/meta.yaml

00:24:45 BIOCONDA ERROR BUILD FAILED recipes/perl-template-toolkit

....
....

00:25:32 BIOCONDA ERROR BUILD SUMMARY: of 9469 recipes, 1 failed and 0 were skipped. Details of recipes and environments follow.
00:25:32 BIOCONDA ERROR BUILD SUMMARY: FAILED recipe recipes/perl-template-toolkit
00:25:32 BIOCONDA ERROR BUILD SUMMARY: SKIPPED recipe perl-bio-phylo-cipres due to failed dependencies ['recipes/perl-template-toolkit']
00:25:32 BIOCONDA ERROR BUILD SUMMARY: SKIPPED recipe perl-bio-phylo-forest-dbtree due to failed dependencies ['recipes/perl-template-toolkit']
00:25:32 BIOCONDA ERROR BUILD SUMMARY: SKIPPED recipe perl-hpc-runner-command due to failed dependencies ['recipes/perl-template-toolkit']
00:25:32 BIOCONDA ERROR BUILD SUMMARY: SKIPPED recipe nbitk due to failed dependencies ['recipes/perl-template-toolkit']
00:25:32 BIOCONDA ERROR BUILD SUMMARY: SKIPPED recipe perl-hpc-runner-command-plugin-logger-sqlite due to failed dependencies ['recipes/perl-template-toolkit']
00:25:32 BIOCONDA ERROR BUILD SUMMARY: SKIPPED recipe perl-bio-viennangs due to failed dependencies ['recipes/perl-template-toolkit']
00:25:32 BIOCONDA ERROR BUILD SUMMARY: SKIPPED recipe perl-bio-phylo due to failed dependencies ['recipes/perl-template-toolkit']
```

Cheers,
Luuk

--- a/recipes/perl-template-toolkit/meta.yaml
+++ b/recipes/perl-template-toolkit/meta.yaml
@@ -10,7 +10,7 @@ source:
   sha256: d161c89dee9b213a7c55709ea782e2dd5923dbd1215b9576612889e6e74a2e06
 
 build:
-  number: 0
+  number: 1
   run_exports:
     - {{ pin_subpackage(name, max_pin='x') }}
 
@@ -43,4 +43,3 @@ about:
 extra:
   additional-platforms:
     - linux-aarch64
-    - osx-arm64

--- a/scripts/check-for-additional-platforms.sh
+++ b/scripts/check-for-additional-platforms.sh
@@ -22,7 +22,7 @@ wget https://github.com/mikefarah/yq/releases/latest/download/yq_${yq_platform}_
 chmod +x ${HOME}/bin/yq
 
 # Find recipes changed from this merge
-files=`git diff --name-only --diff-filter AMR ${git_range} | grep -v 'template' | grep -E 'meta.yaml$' `
+files=`git diff --name-only --diff-filter AMR ${git_range} | grep -E 'meta.yaml$' `
 build=0
 
 for file in $files; do

--- a/scripts/check-for-additional-platforms.sh
+++ b/scripts/check-for-additional-platforms.sh
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -x
+
 # Do not set -x, this script outputs a value with echo
 
 # Check to see if any changed recipes have specified the key

